src/App.js

```diff
@@ -1,12 +1,33 @@
+import {useMemo, useState} from 'react';
 import styled from 'styled-components';
 
+import Categories from './components/categories';
 import ImageContainer from './components/imageContainer';
+import data from './garmentsData';
 
 export default function App() {
+  const [garments] = useState(data);
```

Comment:
even though you don't use the set function yet, I would recommend to add it:
```suggestion
  const [garments, setGarments] = useState(data);
```